templates/components/form/form.jsx

```diff
@@ -66,9 +66,9 @@ var Form = React.createClass({
       input = (<span>{input}<span/></span>);
     }
 
-    var errorTooltip = (
-      <span className="warning">{value.errorMessage}</span>
-    );
+    var errorTooltip = (<span className="warning">{value.customError ? value.customError :
```

Comment:
nested ternary = let's not use nested ternary.
```
var errorTooltip = value.customError;
if (!errorTooltip) {
  errorTooltip = (id === 'password' ? 'Invalid password' : this.getValidationMessages(id)[0]);
}
errorTooltip = <span className="warning">{errorTooltip}<span>;
...
```
for instance? Also, those `==` should have been lint-caught... lastly, can we wrap this hardcoded string in a way that makes it easier to localise later?